aries-site/src/examples/components/select/SelectValidationExample.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import { Box, Button, Form, FormField, Select } from 'grommet';
+
+const options = [
+  'Item One',
+  'Item Two',
+  'Item Three',
+  'Item Four',
+  'Item Five',
+  'Item Six',
+];
+
+export const SelectValidationExample = () => {
+  const [selected, setSelected] = useState('');
+
+  return (
+    <Box width="medium">
+      <Form messages={{ required: 'Required.' }}>
+        <FormField
+          htmlFor="validation-example__input"
+          name="validation-example"
+          label="Label for Required Select"
+          required
+        >
+          <Select
```

Comment:
One thing I noticed is this because the state is on required so there is no focus around the select when its in validation 

https://www.figma.com/file/fZJdRcNrvZ1JjdPRYHpFFU/HPE-Select-Component?node-id=0%3A1

Reply:
I believe that is work-in-progress on Grommet. It would affect all FormFields in this state.

Reply:
oh right I saw that on `textinput` as well. thanks for reminding me!

Reply:
Here's the issue: https://github.com/grommet/grommet/issues/3932